design/work_item_type_group.go

```diff
@@ -0,0 +1,68 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var workItemTypeGroupSigle = JSONSingle(
+	"workItemTypeGroupSigle",
+	`Group of the work-item-types`,
+	workItemTypeGroupData,
+	workItemTypeGroupLinks,
+)
+
+var workItemTypeGroups = JSONList(
+	"workItemTypeGroups",
+	"...",
```

Comment:
removed in https://github.com/fabric8-services/fabric8-wit/pull/1541/commits/c1cb276d949c09b342a53e552407b593bd4f0f5b